docs/book/helpers/doctype.md

```diff
@@ -45,15 +45,32 @@ And then print it out on top of your layout script:
 <?php echo $this->doctype() ?>
 ```
 
-Within an application based off the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton),
-you can specify the doctype via configuration:
+## Usage in a Mezzio Application
+
+The factory `Laminas\View\Helper\Service\DoctypeFactory` checks the application configuration, making it possible to
+define the doctype through your configuration, e.g. `config/autoload/mezzio.global.php`
+or a `ConfigProvider.php` in a module.
+
+```php
+// config/autoload/mezzio.global.php
```

Comment:
Please add the filename before the code example because there are always two parts for this information: what kind of file is this and what is the name or path.

Examples are:

> Create a file for the module configuration called `module.config.php` under `laminas-mvc-tutorial/module/Album/config/`:

> …in a factory, e.g. `src/Album/Controller/AlbumControllerFactory.php`:

> …in a module configuration file, e.g. `module/Album/config/module.config.php`:

In a code comment this would be too long and misplaced.

Thanks in advance!

_(@settermjd and I are still working on documentation writer's guide and we hope to publish it soon.)_

Reply:
done, thanks!

I'm looking forward to a writer's guide! Then hopefully I'll get it right one the first try 🙂